ngraph/python/BUILDING.md

```diff
@@ -120,67 +116,63 @@ cmake .. ^
 ```
 
 There are a couple of things to notice here. One is that the full path to the x64 version of
-MSVC compiler has to be specified. This is because DNNL requires a 64-bit version and cmake may
+MSVC compiler has to be specified. This is because DNNL requires a 64-bit version and `cmake` may
 fail to detect it correctly.
 
-The other equally important thing to note is that the temporary directory where the build is to be installed can be specified.
-If the installation directory is not specified, the default location is `C:\Program Files\OpenVINO`.
-This examples uses `C:\temporary_install_dir` however, a subdirectory of `openvino\build` works as well.
-The final Python wheel will contain the contents of this temporary directory so it's important to set it.
-
 If you want to specify an exact Python version, use the following options:
 ```
 -DPYTHON_EXECUTABLE="C:\Program Files\Python37\python.exe" ^
 -DPYTHON_LIBRARY="C:\Program Files\Python37\libs\python37.lib" ^
 -DPYTHON_INCLUDE_DIR="C:\Program Files\Python37\include" ^
 ```
 
-In order to build and install OpenVINO, build the `install` target:
+In order to build and install OpenVINO™, build the `install` target:
 
     cmake --build . --target install --config Release -j 8
 
-In this step OpenVINO will be built and installed to the directory specified above. You can
+In this step, OpenVINO™ is built and installed to the directory specified above. You can
 adjust the number of threads used in the building process to your machine's capabilities.
 
+Set up the OpenVINO™ environment in order to add a module path to `PYTHONPATH`:
+
+    %OPENVINO_BASEDIR%\openvino_dist\bin\setupvars.bat
+
+#### Build an nGraph Python Wheel
+
 Build the Python wheel package:
```

Comment:
Consider changing the line to the following:

To build the OpenVINO™ as a Python wheel, use the following steps:
1. Build the Inference Engine libraries with CMake using the instructions in the [Configure and Build as a Part of OpenVINO™ Toolkit on Windows](#configure-and-build-as-a-part-of-openvino-toolkit-on-windows) section.
2. Move to the `openvino\ngraph\python` folder in the workspace directory and run the `setup.py` command:

Reply:
Proposed change is not requirement for building Python wheel any more.